hazelcast/src/main/java/com/hazelcast/instance/impl/MainClassNameFinder.java

```diff
@@ -26,11 +26,12 @@
 import java.util.jar.JarFile;
 import java.util.jar.Manifest;
 
-class MainClassFinder {
+class MainClassNameFinder {
```

Comment:
Is there any reason why this class isn't public?

Reply:
It is used from the same package. Public would make it visible to outer world

Reply:
And what's the difference between visibility of `MainClassNameFinder` and `MainMethodFinder` which is public? (I mean why one of them has public and one of them default visibility - they purpose seems to be almost the same)

Reply:
OK. Changed MainMethodFinder too